var/spack/repos/builtin/packages/py-onnxruntime/package.py

```diff
@@ -99,10 +129,28 @@ class PyOnnxruntime(CMakePackage, PythonExtension):
     root_cmakelists_dir = "cmake"
     build_directory = "."
 
+    def patch(self):
+        if self.spec.satisfies("@1.7: + rocm"):
+            filter_file(
+                r"${onnxruntime_ROCM_HOME}/.info/version-dev",
+                "{0}/.info/version".format(self.spec["rocm-core"].prefix),
+                "cmake/CMakeLists.txt",
+                string=True,
+            )
+        if self.spec.satisfies("@1.8: + rocm"):
+            filter_file(
```

Comment:
self.spec.satisfies("@1.18: + rocm"):

Reply:
also, just checking if the version(1.18.0 ) can be  added here ..
 version("1.18.0", tag="v1.18.0", commit="45737400a2f3015c11f005ed7603611eaed306a6")

Reply:
There is already a pr in review to add the newer versions: https://github.com/spack/spack/pull/46329. I was going to wait for that pr to merge first but I've now added the newer versions here to speed things up.

Reply:
Added a note in #46329; if no response you can go ahead and implement here.